src/components/Categories.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import NavBar from './NavBar';
+
+const Books = () => (
+  <>
+    <NavBar />
+  </>
+);
```

Comment:
- [ ] as part of the requirements, the Categories component should display a button with text "Check status" , kindly add that 🙏